resources/schemas/dbscripts/postgresql/targetedms-19.193-19.20.sql

```diff
@@ -0,0 +1,3 @@
+
+
+ALTER TABLE targetedms.Runs ADD COLUMN AuditLogEntriesCount INT4 DEFAULT 0 NOT NULL;
```

Comment:
This will be the first script of 19.20. I believe develop was just recently bumped to set module versions to 19.20. Please make it 19.20-19.21.